test/action.yml

```
@@ -24,8 +24,10 @@ inputs:
       This is used when testing services against the edgexfoundry snap.
       This is useful only when 'snap' input is not set.
     required: false
-  print_logs:
-    description: Deprecated. Logs are uploaded as artifacts.
+  full_config_test:
+    description: |
+      Option for enabling full config options test.
+      This is used when testing services' legacy and mixed options.
```

```suggestion
      Option for enabling full config options test to include for e.g. legacy options testing.
```
The actual behavior can change. So better have this as an example.